JavaScript/YoutubeSearch/styles/styles.css

```diff
@@ -0,0 +1,76 @@
+header {
+  background-color: rgb(153, 23, 23);
+  display: flex;
+  justify-content: center;
+}
+h1 {
+  color: black;
+}
+
+
+.search {
+  display: flex;
+  justify-content: center;
+}
+
+#searchbox {
+  margin: 5%;
+  margin-right: 0;
+  font-size: large;
+  padding: 6px;
+  border-radius: 4%;
+  width: 20%;
+}
+
+button {
+  margin: 5%;
+  margin-left: 0px;
```

Comment:
no need to specify px, when we are using 0.
their is not wrong in specifing, but we have some tools like sonar, which will throw then as code smells.